frontend/src/media/trophy-2.svg

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Generator: Adobe Illustrator 16.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 width="32px" height="32px" viewBox="0 0 32 32" style="enable-background:new 0 0 32 32;" xml:space="preserve">
+<g>
+	<path fill="#B4B4B4" d="M31.734,2.429c-1.438-1.196-3.664-1.953-6.543,0.354c-0.006-0.35-0.01-0.695-0.022-1.059c-1.79,0-16.547,0-18.337,0
+		C6.818,2.088,6.814,2.434,6.81,2.782C3.93,0.477,1.705,1.231,0.268,2.43L0,2.653l0.017,0.346c0.015,0.333,0.432,8.199,4.749,11.769
+		c1.508,1.247,3.305,1.824,5.343,1.701c-0.139,0.438-0.414,1.042-0.731,1.574l1.197,0.718c0.227-0.379,0.602-1.066,0.828-1.769
+		c1.59,1.29,2.982,1.677,2.982,2.631c0,2.148-5.313,3.546-5.504,4.891H8.143v0.567H7.355v5.632h17.116v-5.632h-0.691v-0.567H23.12
+		c-0.188-1.345-5.505-2.741-5.505-4.891c0-0.955,1.393-1.342,2.984-2.631c0.227,0.701,0.601,1.389,0.825,1.768l1.199-0.719
+		c-0.317-0.53-0.595-1.135-0.731-1.572c2.039,0.122,3.834-0.454,5.342-1.701c4.317-3.57,4.733-11.437,4.75-11.77L32,2.65
+		L31.734,2.429z M5.659,13.694C2.301,10.919,1.57,4.828,1.438,3.291c1.586-1.121,3.353-0.673,5.378,1.364
+		C6.824,4.919,6.84,5.169,6.854,5.422c-0.49,0.47-1.151,1.085-1.756,1.633l0.938,1.036c0.36-0.326,0.676-0.613,0.952-0.868
+		c0.409,3.825,1.438,6.242,2.604,7.855C8.095,15.075,6.773,14.615,5.659,13.694z M21.936,29.541H10.219v-2.863h11.716V29.541z
+		 M26.342,13.694c-1.113,0.921-2.436,1.38-3.932,1.384c1.165-1.614,2.195-4.03,2.604-7.854c0.277,0.254,0.592,0.541,0.953,0.867
+		l0.938-1.035c-0.605-0.548-1.268-1.162-1.756-1.635c0.014-0.252,0.027-0.501,0.035-0.766c2.025-2.037,3.793-2.485,5.379-1.365
+		C30.43,4.827,29.701,10.918,26.342,13.694z"/>
+</g>
+<g>
+</g>
+<g>
```

Comment:
do we need these empty g's here? (and in other files?)

Reply:
done [here](https://github.com/salsita/foosball-rating/pull/92/commits/2c682cc5476a670cf83a5400ca8574b62126ce51#diff-42a4e71be5434d1df2e159d77895f5ab) and below